src/plugins/slisp/main.js

```diff
@@ -0,0 +1,68 @@
+const slisp = require("./slisp.js");
+
+/**
+ * A one stop shop for all things a Spike Plugin could need!
+ */
+const spikeKit = require("../../spikeKit.js");
+
+/**
+ * The display name of the plugin.
+ */
+const NAME = "Slisp";
+/**
+ * Slug used to programmatically refer to the plugin. Lowercase letters, numbers, and dashes only.
+ */
+const SLUG = "slisp";
+/**
+ * The author(s) of this plugin.
+ */
+const AUTHOR = "Joshua Maxwell";
+/**
+ * Commands supported by this plugin. Do not include the prefix.
+ * Good: command
+ * Bad: $command
+ */
+const COMMANDS = ["slisp"];
+
+/**
+ * Handles help requests for this plugin.
+ * @param {string} prefix The command prefix.
+ * @param {string} command The command issued, without the prefix.
+ * @param {string} args The rest of the message.
+ * @returns Help text to be sent back to the user.
+ */
+function help(prefix, command, args) {
+  return "How to use the command with given args.";
```

Comment:
This needs to be filled out properly for the command. Could just be a link to the wiki.